admin_guide/configuring_openstack.adoc

```diff
@@ -0,0 +1,75 @@
+= Configuring for OpenStack
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+When deployed on
+link:https://www.openstack.org/[OpenStack], OpenShift can be configured to
+access OpenStack infrastructure and e.g. use
+link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-openstack-platform/version-7/red-hat-enterprise-linux-openstack-platform-7-architecture-guide/chapter-1-components#comp-cinder[OpenStack Cinder]
+volumes as persistent storage for application data.
+
+=== Cloud configuration
+As an administrator, create *_/etc/cloud.conf_* file on all your OpenShift
+machines, both masters and nodes:
+
+----
+[Global]
+auth-url = <OS_AUTH_URL>
+username = <OS_USERNAME>
+password = <password>
+tenant-id = <OS_TENANT_ID>
+region = <OS_REGION_NAME>
+
+[LoadBalancer]
+subnet-id = <UUID of the load balancer subnet>
+----
+
+Consult your OpenStack administrators for values of `*OS_*` variables, they are
+commonly used in OpenStack world.
```

Comment:
Suggestion:

"Consult your OpenStack administrators for values of the OS_ variables, which are commonly used in the OpenStack world."